packages/path_provider/path_provider_windows/pubspec.yaml

```diff
@@ -21,7 +21,7 @@ dependencies:
     sdk: flutter
   path: ^1.8.0
   path_provider_platform_interface: ^2.0.0
-  win32: ^2.1.0
+  win32: ^3.0.0
```

Comment:
If this package works fine with `^2.1.0` and `^3.0.0` then we shouldn't bump the minimum dependency but rather set it to `>=2.1.0 <4.0.0`.